mergify_engine/mergify_pull.py

```diff
@@ -168,6 +169,42 @@ def _get_checks(self):
                                for s in self._get_statuses()])
         return generic_checks
 
+    def _get_team_members(self, name):
+        organization, _, team_slug = name.partition("/")
+        if organization[0] != "@" or not team_slug or '/' in team_slug:
+            # Not a team slug
+            return
```

Comment:
I'd say rename this method to `_resolve_login` and change this line to `return [name]`.
You can then simplify `resolve_teams`.